### PR TITLE
 RHTAP-1201: Add RH cluster secrets stores

### PR DESCRIPTION
**Pull request by gbenhaim**
The new stores will be used for fetching secrets to the RH clusters.

### CODE REVIEW

File: components/cluster-secret-store-rh/base/appsre-stonesoup-vault-rh-secret-store.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      version: v2
+      auth:
+        # VaultAppRole authenticates with Vault using the
+        # App Role auth mechanism
+        # https://www.vaultproject.io/docs/auth/approle
+        appRole:
+          # Path where the App Role authentication backend is mounted
+          path: approle
+          # RoleID configured in the App Role authentication backend
+          roleId: d95dc71a-c0e7-158d-f4e5-75dd5c5c5557
```
Comment (bamachrn) [inline components/cluster-secret-store-rh/base/appsre-stonesoup-vault-rh-secret-store.yaml:22]: do we want to hardcode it in the base. Or putting values from overlays make more sense, like the bucket writer?

Comment (gbenhaim) [reply components/cluster-secret-store-rh/base/appsre-stonesoup-vault-rh-secret-store.yaml]: fixed

**Comment by ernesgonzalez33** [comment]
lgtm

**Comment by gbenhaim** [comment]
The CI fails because of a known issue no related to my pr RHTAPBUGS-739

**Pull request merged**